tests/functional_tests/ffront_tests/test_interface.py

```diff
@@ -216,6 +200,15 @@ def bool_or(a: Field[..., "bool"], b: Field[..., "bool"]):
         _ = FieldOperatorParser.apply_to_function(bool_or)
 
 
+def test_shift():
+    Offset = 0  # for parsing it doesn't matter what this object is
+
+    def shifts(a: Field[..., "bool"]):
+        return a(Offset[1]) + a(Offset[-1])  # TODO we should test these cases separate
+
+    _ = FieldOperatorParser.apply_to_function(shifts)
+
```

Comment:
This test would pass even if the parser were to simply ignore and drop offsets. Ideally we would also add a negative offset case to `test_lowering.test_shift`. But if we have that, we don't really need this one.

Reply:
removed in favor of a lowering test